Packs/CortexResponseAndRemediation/Playbooks/playbook-WmiPrvSe.exe_Rare_Child_Command_Line.yml

```diff
@@ -0,0 +1,1365 @@
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      closeNotes:
+        simple: The alert has been resolved, confirmed as a True Positive
+      closeReason:
+        simple: Resolved - Handled by the playbook "WmiPrvSe.exe Rare Child Command Line" as True Positive
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 172,
+          "y": 3007
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: 7f02a14e-8ff0-4bb6-860b-17d2471ce868
+    type: title
+    task:
+      id: 7f02a14e-8ff0-4bb6-860b-17d2471ce868
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 172,
+          "y": 3177
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: 7122a5a1-429a-4ba1-8069-c7d736845fa1
+    type: regular
+    task:
+      id: 7122a5a1-429a-4ba1-8069-c7d736845fa1
+      version: -1
+      name: Close the Alert as False Positive
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      closeNotes:
+        simple: The alert has been resolved, confirmed as a False Positive
+      closeReason:
+        simple: Resolved - Handled by the playbook "WmiPrvSe.exe Rare Child Command Line" as False Positive
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1140,
+          "y": 3007
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: 095c11b6-e83f-49f8-8761-24fe79b5d968
+    type: condition
+    task:
+      id: 095c11b6-e83f-49f8-8761-24fe79b5d968
+      version: -1
+      name: Approval Required – Malicious Activity Detected
+      description: |-
+        **Recommendation: Isolation Required – Malicious Activity Detected**
+
+        The detection meets high-confidence criteria for malicious activity based on the following conditions:
+
+        **Matched Verdicts:**
+        * Matches for high-risk command line analysis profiles: mixed_case_powershell, double_encoding, amsi_techniques, malicious_commands, or powershell_suspicious_network_patterns.
+
+        * Malicious IP address detected.
```

Comment:
Malicious IP address detected. Not checked in condition #19.
Please update the description.